examples/gno.land/p/demo/accesscontrol/accesscontrol_test.gno

```diff
@@ -0,0 +1,221 @@
+package accesscontrol
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/testutils"
+)
+
+// TestAccessControl verifies the access control functionality.
+func TestAccessControl(t *testing.T) {
+	admin := testutils.TestAddress("admin")
+	user1 := testutils.TestAddress("user1")
+	user2 := testutils.TestAddress("user2")
+
+	// Create new RoleData
+	RoleData := NewRoleData("admin", admin)
```

Comment:
This is a nit pick but making a variable name the same as the type name makes the code a bit harder to read.  Perhaps naming this `roleData` would be better.

Reply:
https://github.com/gnolang/gno/pull/2307/commits/dadcc5dc4352a18cc605148f417aa195e8cb01af